### PR TITLE
add airbnb eslint

### PR DESCRIPTION
**Pull request by Baelfire18**
## What?

- Add airbnb eslint
- Add eslint commands
- Autocorrect eslint
- Correct eslint errors
- Add eslint rules

## Why?

To standardize code rules

## How?

Installing the package, running the commands and fixing if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
 
-# Use
+## Use
 
 Just star this repository and go to the website.
 
-# Contribute
+## Linter
+
+This proyect uses the airbnb eslint style guide. For runing it, you can use the following command:
```
Comment (nmassardot) [inline README.md:11]:
```suggestion
This proyect uses the airbnb eslint style guide. For running it, you can use the following command:
```

File: README.md [hunk 2]
```diff
@@ -2,17 +2,24 @@
...
+
+```bash
+yarn lint # To check linting
```
Comment (nmassardot) [inline README.md:14]:
```suggestion
yarn lint # Check linting
```

File: README.md [hunk 3]
```diff
@@ -2,17 +2,24 @@
...
+yarn lint --fix # To fix autocorrectable linting
```
Comment (nmassardot) [inline README.md:15]:
```suggestion
yarn lint --fix # Automatic fix linting
```

File: src/App.test.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+/* eslint-disable no-undef */
+test('renders home page', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/Who doesn't follow you back/i);
+  expect(linkElement).toBeInTheDocument();
+});
+/* eslint-enable no-undef */
```
Comment (nmassardot) [inline src/App.test.jsx:11]: Delete this file plz, no test in here

File: src/components/SectionFollowers.jsx [hunk 5]
```diff
@@ -156,29 +159,33 @@ function SectionFollowers({ username }) {
...
+              className={clsx('text-3xl font-bold', 'mb-2', 'text-center')}
             >
               Remember to star the repository to see whom doesn't follow you back!
             </h1>
-            <p className={clsx("text-md", "mb-8")}>
-              If you think something's wrong, {" "}
+            <p className={clsx('text-md', 'mb-8')}>
+              If you think something's wrong,
+              {' '}
+              {' '}
```
Comment (nmassardot) [inline src/components/SectionFollowers.jsx:169]: Delete the double space

File: src/components/logos/GithubLogo.jsx [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
+import React from 'react';
 
-function GithubLogo(props) {
+function GithubLogo({ className }) {
   return (
     <svg
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 24 24"
-      {...props}
+      className={className}
```
Comment (nmassardot) [inline src/components/logos/GithubLogo.jsx:8]: I prefer this with props, in case I need something else

**Review by nmassardot — CHANGES_REQUESTED** [review]
Change the rule from single quotes to double quotes please
Thanks for your contribution!

**Comment by Baelfire18** [comment]
All changes requestes (including erasing the test 😢) done!
Also resolved conflict, and it also passes the linter

**Review by nmassardot — APPROVED** [review]

**Pull request merged**